README.md

```diff
@@ -9,17 +9,22 @@
 
 # AtlasReader
 
-This package provides a Python interface for generating coordinate tables and region labels from statistical MRI images.
+This package provides a Python interface for generating coordinate tables and
```

Comment:
What's our markdown linting rule? 

I pretty strongly prefer sentences on separate lines because I think it make it intuitive to see what has changed....but this is just a question, no need to fix in this PR.

Reply:
I'm up for any rule. At the moment, I decided to have line breaks after 80 chars. But one sentence per line is also fine to me.